go/teams/teams.go

```diff
@@ -547,50 +547,50 @@ func (t *Team) getAdminPermission(ctx context.Context, required bool) (admin *SC
 	return &ret, nil
 }
 
-func (t *Team) changeMembershipSection(ctx context.Context, req keybase1.TeamChangeReq) (SCTeamSection, *PerTeamSharedSecretBoxes, *memberSet, error) {
+func (t *Team) changeMembershipSection(ctx context.Context, req keybase1.TeamChangeReq) (SCTeamSection, *PerTeamSharedSecretBoxes, map[keybase1.TeamID]*PerTeamSharedSecretBoxes, *memberSet, error) {
```

Comment:
I could make a struct out of this one too, though it's only called in one place in production code. I could reuse the `sigPayloadArgs` struct as a return value here, but only some of the fields of it would be filled. I'm torn.